Remotion/Security/UnitTests/Metadata/EnumValueInfoListContentsConstraintTest.cs

```diff
@@ -55,25 +55,24 @@ public void AssertWithValidValue ()
     public void AssertWithListEmpty ()
     {
       var constraint = new EnumValueInfoListContentsConstraint ("First");
-      Assert.That (constraint.Matches (new List<EnumValueInfo> ()), Is.False);
+      Assert.That (constraint.ApplyTo (new List<EnumValueInfo> ()).IsSuccess, Is.False);

```

Comment:
It's not really part of this task, so perhaps a follow-up, but it might be a good idea to add tests for the other constraint types? @Stefan-Ilic